code/modules/paperwork/fax.dm

```diff
@@ -299,6 +342,24 @@
 		return TRUE
 	return FALSE
 
+/**
+ * The procedure for sending a paper to virtual admins fax machine.
+ *
+ * This procedure is similar to the send procedure except that it sends the paper to
+ * a "virtual" fax to a special administrator list.
+ * Arguments:
+ * * paper - The paper to be sent.
+ * * sender - Reference to the sender's substance.
+ * * receiver_name - The recipient's fax name, which will be displayed in the administrator's list.
+ * * receiver_color - The color the receiver_name will be colored in.
+ */
+/obj/machinery/fax/proc/send_to_additional_faxes(obj/item/paper/paper, mob/sender, receiver_name, receiver_color)
+	GLOB.fax_manager.receive_request(sender, src, receiver_name, paper, receiver_color)
+	history_add("Send", receiver_name)
+	INVOKE_ASYNC(src, .proc/animate_object_travel, paper, "fax_receive", find_overlay_state(paper, "send"))
+	playsound(src, 'sound/machines/high_tech_confirm.ogg', 50, FALSE)
```

Comment:
I don't like the copy paste here with send(), can you make them share one sending proc for this?